packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/optimize-ciemss-operation.ts

```diff
@@ -38,6 +44,7 @@ export interface ConstraintGroup {
 
 export interface OptimizeCiemssOperationState extends BaseState {
 	// Settings
+	startTime?: number;
```

Comment:
From my understanding we are not allowing the users to update the start time of their simulation run.
Has this changed? Why do we want to store this?

Reply:
Moved to the `InterventionPolicy` interface as we discussed